alacritty/src/renderer/mod.rs

```diff
@@ -421,6 +421,10 @@ struct InstanceData {
     bg_a: f32,
     // Flag indicating that glyph uses multiple colors, like an Emoji.
     multicolor: u8,
+    // Set to 1 if glyph is a WIDE_CHAR.
+    is_wide_char: u8,
+    // Set to 1 if glyph is a WIDE_CHAR_SPACER.
+    is_spacer: u8,
```

Comment:
I guess we can reuse multicolor value and call it like cell_flags and just pass all three values in it. Having 3 boolean flags to pass to GPU is wasteful to me.

Reply:
Ok, let me have a try.